cinema_service/urls.py

```diff
@@ -2,10 +2,33 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularRedocView,
+    SpectacularSwaggerView
+)
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
-    path("api/user/", include("user.urls", namespace="user")),
+    path(
+        "api/user/",
+        include(
+            "user.urls",
+            namespace="user"
+        )
+    ),
     path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
Ensure that the `debug_toolbar` is only included in development environments. You can conditionally include this path based on the `DEBUG` setting to prevent exposing debugging information in production.